documentation/src/main/markdown/troubleshooting/commonwindowsintricacies.md

```diff
@@ -0,0 +1,4 @@
+# Windows Intricacies
+
+* Windows does not support passing spaces in arguments which are enclosed within double quotes as it will consider space as a separator for arguments and discard all the values which would be within double quotes. Eg: If you pass --detect.project.name=" Windows Project ", then that is being interpreted as "--detect.project.name=" "Windows" "Project". So, if you want to pass spaces inside any argument in Windows OS, then you can either use single quotes ('single') or you can use backtick character (\`) to escape spaces.
```

Comment:
Suggestion:
* Windows considers space as a separator for arguments and discards all the values which would be within double quotes. Eg: If you pass --detect.project.name=" Windows Project ", then that is being interpreted as "--detect.project.name=" "Windows" "Project". To pass spaces inside an argument in Windows OS, then you can either use single quotes ('single') or you can use backtick character (\`) to escape spaces.
* Using Windows with multibyte characters for different languages such as Korean or Japanese, you must configure Windows by using the chcp command to change the character code for cmd shell.

Reply:
Will update according to suggestion.